_config/site.yml

```diff
@@ -211,17 +211,10 @@ profiles:
       <<: *deploy
       path: /stg_htdocs/it-rhd-stg-main
 
-  # Octopus integration configuration
```

Comment:
Just checking since this isn't strictly a protocol change, but is this removal okay?